packages/client/tests/functional/_utils/providers.ts

```diff
@@ -23,6 +23,8 @@ export enum RelationModes {
   PRISMA = 'prisma',
 }
 
+export type GeneratorTypes = 'prisma-client-js' | 'prisma-client-ts'
```

Comment:
Fine to keep it as is for consistency with other types in this file, but they all should be changed to singular

Reply:
Yeah we could also all change them away from being ts enums I think? 🤔

Reply:
I'll put it on my cleanup list for during the flight next week 😅